pkg/api/org_users.go

```diff
@@ -29,6 +29,7 @@ func addOrgUserHelper(cmd models.AddOrgUserCommand) response.Response {
 
 	userQuery := models.GetUserByLoginQuery{LoginOrEmail: cmd.LoginOrEmail}
 	err := bus.Dispatch(&userQuery)
+
 	if err != nil {
```

Comment:
Nit, but co-locating error checks with the error assignment helps readability:
```suggestion
	err := bus.Dispatch(&userQuery)
	if err != nil {
```